DependencyInjection/MonologExtension.php

```diff
@@ -468,6 +468,8 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 $handler['bubble'],
                 $handler['use_ssl'],
                 $handler['message_format'],
+                $handler['host'],
```

Comment:
I tihnk you can just pass 'api.hipchat.com' as value for the host here in case it's null, that would solve the need for a change in monolog